Packs/GenericSQL/TestPlaybooks/generic-sql-oracle.yml

```diff
@@ -0,0 +1,362 @@
+id: generic-sql-oracle
+version: -1
+contentitemexportablefields:
```

Comment:
I think running demisto-sdk format will remove this section and other un-needed stuff.